deploy/crds/mellanox.com_macvlannetworks_crd.yaml

```diff
@@ -0,0 +1,92 @@
+# Copyright 2020 NVIDIA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: macvlannetworks.mellanox.com
+spec:
+  group: mellanox.com
+  names:
+    kind: MacvlanNetwork
+    listKind: MacvlanNetworkList
+    plural: macvlannetworks
+    singular: macvlannetwork
+  scope: Cluster
```

Comment:
why is it cluster scoped ? i thought we are doing the same pattern as sriov-network-operator ?

Reply:
The macvlan passes the desired namespace for the net-attach-def through the NetworkNamespace which allow to deploy it in any desired namespace.
Is there any use case for MacvlanNetowrk to be namespace scope?

Reply:
Having macvlannetwork cluster scope allows setting controller reference for net-attach-def CRD.